luarocks.yaml

```
@@ -0,0 +1,41 @@
+package:
+  name: luarocks
+  version: 3.11.1
+  epoch: 0
+  description: "Package manager for Lua modules"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - luajit-dev
+      - openssl-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/luarocks/luarocks
+      tag: v${{package.version}}
+      expected-commit: fa2736e7c68094710643577b0153dac72ce4bcdc
+
+  - runs: |
+      ./configure --prefix=/usr --with-lua=/usr
+      make build
+      DESTDIR="${{targets.destdir}}" make install
```

```suggestion
  - runs: ./configure --prefix=/usr --with-lua=/usr
  - uses: autoconf/make
  - uses: autoconf/make-install
```
the host flag is something I was not able to hack on. Even passing `""` doesn't make the `autoconf/configure` pipeline happy. 

One other thing is that `apk add luarocks` in the current state pulls in fluentbit which is something I think is not required by this package. 💭